packages/backend/src/apps/postman-sms/actions/send-sms/index.ts

```diff
@@ -0,0 +1,95 @@
+import type { IRawAction } from '@plumber/types'
+
+import { ZodError } from 'zod'
+
+import { postmanSmsConfig } from '@/config/app-env-vars/postman-sms'
+import StepError, { GenericSolution } from '@/errors/step'
+import { ensureZodObjectKey, firstZodParseError } from '@/helpers/zod-utils'
+
+import getDataOutMetadata from './get-data-out-metadata'
+import { fieldSchema, postmanMessageSchema } from './schema'
+
+const action = {
+  name: 'Send SMS',
+  key: 'sendSms',
+  description: 'Sends an SMS; display name is set to plumber.gov.sg',
```

Comment:
```Sends an SMS under Gov.SG sender ID```
in future when credentials are added, it'll no longer be under plumber.gov.sg

Reply:
done